torch/csrc/distributed/c10d/symm_mem/CUDASymmetricMemoryOps.cu

```diff
@@ -112,9 +115,9 @@ void init_elementwise_launch_config(
 
   if (numel_per_split <= max_num_threads * numel_per_thread) {
     num_blocks = 1;
-    num_threads = at::round_up(
-        at::ceil_div(numel_per_split, numel_per_thread),
-        static_cast<size_t>(at::cuda::warp_size()));
+    num_threads = at::ceil_div(numel_per_split, numel_per_thread);
+    num_threads = max(num_threads, MAX_CUDA_P2P_DOMAIN_SIZE);
```

Comment:
Better to pass world_size here and make max equal to world size

Reply:
Thanks. Fixed :)